### PR TITLE
OPSEXP-2293: use ATS common named templates in ai & search ent charts

### PR DESCRIPTION
**Pull request by alxgomz**
Ref: OPSEXP-2293

### CODE REVIEW

File: charts/alfresco-search-enterprise/templates/_helpers-ats.tpl [hunk 1]
```diff
@@ -17,7 +17,7 @@ Usage: include "alfresco-search-enterprise.sfs.fullurl" "URL"
 
 */}}
 {{- define "alfresco-search-enterprise.sfs.fullurl" -}}
```
Comment (gionn) [inline charts/alfresco-search-enterprise/templates/_helpers-ats.tpl:19]: I guess you can drop these two named templates and inline them given they are used only in a single place and without any additional wrapping logic

Comment (alxgomz) [reply charts/alfresco-search-enterprise/templates/_helpers-ats.tpl:19]: I could but I decided to not do it to avoid a major bump

Comment (gionn) [reply charts/alfresco-search-enterprise/templates/_helpers-ats.tpl:19]: why major bump? those named templates are meant to be used internally and we are not changing any behavior (hopefully)

Comment (alxgomz) [reply charts/alfresco-search-enterprise/templates/_helpers-ats.tpl:19]: they are used [here](https://github.com/Alfresco/acs-deployment/blob/2c0e32ba03727e4a3f5294d65879edfadf8a42a6/helm/alfresco-content-services/templates/config-infrastructure.yaml#L59)

**Review by gionn — APPROVED** [review]

**Pull request merged**